wpeverest/everest-forms
# Feature - Anti-spam honeypot form field for spam protection.

### All Submissions:






### Changes proposed in this Pull Request:



You won’t see it in the embedded form at all unless you dig into the code. This is because a honeypot field isn’t meant to be visible to regular users — instead, only bots should see it. Then when a bot fills out this field, that’s how we know they’re not human and are spambots.

### How to test the changes in this Pull Request:

1. Edit the form and **Enable anti-spam honeypot** option.
2. Preview the form in the frontend, there should exist honeypot container.
3. Edit and add some value in the honeypot field, OR try any spambots also.
4. Try to submit the form, entry is not received and a honeypot log is created :)

### Types of changes:




### Other information:




### Changelog entry

> Feature - Anti-spam honeypot form field for spam protection.
